frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/actions/option_action.dart

```diff
@@ -20,24 +21,24 @@ enum OptionAction {
   divider,
   align;
 
-  String get assetName {
+  FlowySvgData get svg {
     switch (this) {
       case OptionAction.delete:
-        return 'editor/delete';
+        return FlowySvgs.delete_editor;
       case OptionAction.duplicate:
-        return 'editor/duplicate';
+        return FlowySvgs.duplicate_editor;
       case OptionAction.turnInto:
-        return 'editor/turn_into';
+        return const FlowySvgData('editor/turn_into');
       case OptionAction.moveUp:
-        return 'editor/move_up';
+        return const FlowySvgData('editor/move_up');
       case OptionAction.moveDown:
-        return 'editor/move_down';
+        return const FlowySvgData('editor/move_down');
       case OptionAction.color:
-        return 'editor/color';
+        return const FlowySvgData('editor/color');
       case OptionAction.divider:
-        return 'editor/divider';
+        return const FlowySvgData('editor/divider');
       case OptionAction.align:
-        return 'editor/align/center';
+        return const FlowySvgData('editor/align/center');
```

Comment:
cc @annieappflowy, it appears that some of the svgs are missing for turn_into, move_up, move_down, color, and divider.

Reply:
I'm not sure what these are defaulting to right now...